docs/content/quickstart.md

```diff
@@ -5,52 +5,92 @@ date = "2022-05-08T14:05:02.118466Z"
+
+## Getting the `bart` and `spin` Binaries
 
 First, you need the `bart` and `spin` binaries configured in your path.
-For Spin, follow [the Spin quickstart guide](https://spin.fermyon.dev/quickstart)
-or head over to GitHub and download [the latest canary release](https://github.com/fermyon/spin/releases/tag/canary).
 
-For `bart`, the Bartholomew CLI, either [follow the contribution guide](./contributing.md)
-and build it from source, or download the pre-built binary from [the latest
-GitHub release](https://github.com/fermyon/bartholomew/releases/).
+### Spin
+
+For Spin, follow [the Spin quickstart guide](https://spin.fermyon.dev/quickstart) which details how to either:
+- download the latest Spin binary release,
+- clone and install Spin using cargo, or
+- clone and build Spin from source.
 
-## Creating your first website with Bartholomew
+### Bartholomew CLI
 
-Now that you have the `bart` CLI in your path, you can use it to create your
-first website from a template:
+For the `bart` CLI, there are two options:
+- download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/), or
+- clone and install `bart` from source, using the following commands:
 
 ```bash
-# create a new site in the `website/` directory, based on the default Bartholomew site template
-$ bart new site --git https://github.com/fermyon/bartholomew-site-template website
-Successfully created new Bartholomew website in directory website.
-Run spin up --file website/spin.toml to start your website locally.
+$ git clone https://github.com/fermyon/bartholomew.git
+$ cd bartholomew
+$ make bartholomew
 ```
 
-This command will pull the git template locally, and you can start Spin and run
-your new website locally:
+## Spin Templates
+
+This quickstart method uses a Bartholomew website template. So while we do require `spin` and `bart` CLI (as per the details above) everything else we need (to launch our Bartholomew CMS website) is packaged up in the Bartholomew website template. Including the `bartholomew.wasm` and the `spin_static_fs.wasm` files which take care of Bartholomew's business logic and [Spin's file server](https://github.com/fermyon/spin-fileserver) needs, respectively. We will start working with the template in the next section.
+
+## Create Your First Website With Spin
+
+Now that you have the `spin` in your path, you can use it to install the Bartholomew site template, which we spoke about earlier:
 
 ```bash
-$ cd website
-$ spin up --follow-all
+$ spin templates install --git https://github.com/fermyon/bartholomew-site-template
```

Comment:
I think this is a Bartholomew site template, not a Spin template. At least the command fails when I try to run this.
```
🐟 > spin templates install --git https://github.com/fermyon/bartholomew-site-template
Copying remote template source
The specified source contained no templates
```

Reply:
I see Ivan's comments below. In this case if the bartholomew-website-template is not a Spin application/component template then I can just clone the template and issue the Spin commands from inside that clone.